### PR TITLE
meilisearch: fix checksum

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by SMillerDev** [comment]
Did you open an upstream issue for this?

**Comment by chenrui333** [comment]
not yet, let me raise it now.

File: Formula/meilisearch.rb [hunk 1]
```diff
@@ -2,8 +2,9 @@ class Meilisearch < Formula
   desc "Ultra relevant, instant and typo-tolerant full-text search API"
   homepage "https://docs.meilisearch.com/"
   url "https://github.com/meilisearch/MeiliSearch/archive/v0.25.2.tar.gz"
-  sha256 "5097f0c6de042125561e669a1d9d09c4ee6425e9715fd0ad59dabeeef1b99ef8"
+  sha256 "544f041c3b8e5217b2144d825453b05887c15a482296735e0a3fcaf070669db4"
```
Comment (chenrui333) [inline Formula/meilisearch.rb:5]: They have confirmed this is the right one at the moment.

**Comment by carlocab** [comment]
I don't think this is ready to merge since upstream don't have an explanation for the checksum mismatch either.

**Comment by cho-m** [comment]
Closing as new version merged in #96820

**Pull request closed without merging**